reference/commands/output/search.rst

```diff
@@ -37,46 +37,44 @@ The output JSON contains a two first level keys:
 
   .. code-block:: json
 
-      {
-          "error":false,
-          "results":[
-              {  
-                  "remote":"conan-center",
-                  "items":[
-                      {
-                          "recipe":{
-                              "id":"eigen/3.3.4@conan/stable"
```

Comment:
eigen is not in conan-center-index yet?

Reply:
yes, it's there. but take a look again, it's about search output, we have both from CCI and Conan Community in conan center, so the output contains both. When I was reviewing my changes, I got confused at same point haha!